Dockerfile

```diff
@@ -225,6 +225,18 @@ RUN mkdir -p /vmtoolsd/${LIBDNET} &&\
     make &&\
     make install && make DESTDIR=$ROOTFS install
 
+# Download and build Parallels Tools
+ENV PRL_TOOLS_URL = http://download.parallels.com/desktop/v11/11.0.0-rtm/ParallelsTools-11.0.0-30901-boot2docker.tar.gz
```

Comment:
Does this syntax actually work properly with spaces around the `=` like this?

If so, I think someone needs to get @duglin on the line and make him pay for his crimes... :wink: (just joking @duglin :heart:)

In seriousness, I'd rather see this file stay at least somewhat self-consistent; what's the justification for having this as one long URL that's going to be "fun" to update instead of just the relevant "version" parts?

```Dockerfile
ENV PRL_MAJOR 11
ENV PRL_VERSION 11.0.0
ENV PRL_REVISION 30901

RUN mkdir -p /prl_tools && curl -L "http://download.parallels.com/desktop/v${PRL_MAJOR}/${PRL_VERSION}-rtm/ParallelsTools-${PRL_VERSION}-${PRL_REVISION}-boot2docker.tar.gz" ...
```